api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -22,6 +22,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // This configuration allows the built-in LEAST_REQUEST LB policy to be configured via the LB policy
 // extension point. See the :ref:`load balancing architecture overview
 // <arch_overview_load_balancing_types>` for more information.
+// [#next-free-field: 6]
 message LeastRequest {
   // The number of random healthy hosts from which the host with the fewest active requests will
   // be chosen. Defaults to 2 so that we perform two-choice selection if the field is not set.
```

Comment:
This PR change the behavior of the LB. It's necessary to add some more comment to tell that if the `choice_count` is larger than or equal to the hosts size, a full scan will be used.

Reply:
This behavior change should be harmless to end users and maybe make the LB's behavior closer to it's name `least request`. 
So, I think a change log should enough rather than a runtime guard.

But still need a check from @envoyproxy/runtime-guard-changes